resources/templates/nav/nav-top.php

```diff
@@ -10,8 +10,56 @@
     <h2 id="block-mainnavigation-menu">Main navigation</h2>
 	<?php
 		wp_nav_menu( array(
-			'theme_location' => 'common_design_top',
-			'menu_class' => 'wp-menu-item',
+			'container'     => 'ul',
+			'theme_location' => 'top',
```

Comment:
@anna-prince 
Changing the location from `common_design_top` to `top` makes the menu work as we'd expect (unlike yesterday when certain menu items weren't showing).
To correspond with https://github.com/UN-OCHA/common-design-wordpress/blob/cd-289-Wordpress-Starter-Theme/src/navs.php#L22
https://codex.wordpress.org/Navigation_Menus